### ISSUE TITLE
eprload can't load files

### ISSUE THREAD

**Issue by spribitzer** [StollLab/EasySpin#197]
Not sure of this is a bug that exists only with MATLAB 2020a, but I noticed that I'm having troubles loading files with with `eprload`.
Regardless of whether I'm loading a file that is in the working directory or a different directory, I get an error message: 

Here is a MWE: 
```matlab
% any of the following ways to provide the file path gives an error message:
FilePath = "408923.DTA";
FilePath = ""./data/408923.DTA""
FilePath = "408923";
FilePath = join(["./data/" "408923" ".DTA"],'')

[t, V] = eprload(FilePath);
```

The error message is 
```matlab
Error using exist
The first input to exist must be a string scalar or character
vector.

Error in eprload (line 96)
LocationType = exist(FileName,'file');
```

Apparently the string array thats created in line 95 in `eprload`
```matlab
FileName = [FullBaseName FileExtension];
 ```
is no longer processed correct `exist` and we need to switch it to:
```matlab
FileName = join([FullBaseName FileExtension],'');
```

**Comment by spribitzer** [comment]
Just changing 
```matlab
FileName = [FullBaseName FileExtension];
```
alone does not solve the entire problem. Another issue is
``` matlab
if isempty(FileExtension)
  if exist(join([FullBaseName '.dta'],''),'file'), FileExtension = '.dta'; end
  if exist(join([FullBaseName '.DTA'],''),'file'), FileExtension = '.DTA'; end
  if exist(join([FullBaseName '.spc'],''),'file'), FileExtension = '.spc'; end
end
```
since `exist(join([FullBaseName 'xxx'],''),'file')` returns `2` if the file exists.

**Comment by stestoll** [comment]
The fundamental issue here is that you are feeding `eprload` with a string scalar instead of a character vector. The two should be handled equivalently, but EasySpin is not designed or tested to handle string scalars, which are a relatively recent addition to MATLAB. We need to add corresponding tests for `eprload` and other functions.

**Comment by stestoll** [comment]
Fixed for `eprload` in commit 34f1332932b4789c7344d9a8517dd404513eb3ab
